api/src/main/java/com/debugmafia/clueless/model/Player.java

```diff
@@ -0,0 +1,52 @@
+package com.debugmafia.clueless.model;
+
+import java.util.Set;
+
+public class Player {
+
+  private String userName;
+  private int playerNum;
```

Comment:
Is `playerNum` used for determining turn order?

Reply:
More so for who is player 1, to be able to start the game. That was my thought anyway.